### PR TITLE
Improve getting started documentation for "Your first 2D game" tutorial

### PR DESCRIPTION
**Pull request by BartlomiejSzczotka**
Improve documentation for "Your first 2D game" tutorial -> making it more accurate

### CODE REVIEW

File: getting_started/first_2d_game/05.the_main_game_scene.rst [hunk 1]
```diff
@@ -181,6 +181,10 @@ the other two timers. ``ScoreTimer`` will increment the score by 1.
         GetNode<Timer>("ScoreTimer").Start();
     }
 
+After connecting the ``timeout()`` signal for each Timer nodes, Your scene should look like below (see connection icon next to each node):
```
Comment (AThousandShips) [inline getting_started/first_2d_game/05.the_main_game_scene.rst:184]:
```suggestion
After connecting the ``timeout()`` signal for each Timer node,
your scene should look like below (see connection icon next to each node):
```

File: getting_started/first_2d_game/img/timer_signal.webp
Comment (AThousandShips) [inline getting_started/first_2d_game/img/timer_signal.webp:1]: Why is there a yellow splotch over part of the image? Is this an artifact of compression or something?

Comment (BartlomiejSzczotka) [reply getting_started/first_2d_game/img/timer_signal.webp:1]: no -> I just mark it

Comment (AThousandShips) [reply getting_started/first_2d_game/img/timer_signal.webp:1]: It's not very clear, so I'd remove it or make it more visible and organized, it's very haphazard

Comment (BartlomiejSzczotka) [reply getting_started/first_2d_game/img/timer_signal.webp:1]: sure! Thx!

Comment (AThousandShips) [reply getting_started/first_2d_game/img/timer_signal.webp:1]: The yellow part is still there, please remove or make it clearer

File: getting_started/first_2d_game/05.the_main_game_scene.rst (hunk 1 above)
Comment (AThousandShips) [reply getting_started/first_2d_game/05.the_main_game_scene.rst:184]: You still haven't fixed this

**Comment by skyace65** [comment]
Closing this for not updating the PR based on AThousandShips review

**Pull request closed without merging**